src/Common/src/TypeSystem/IL/Stubs/DelegateMarshallingMethodThunk.cs

```diff
@@ -123,7 +139,7 @@ public override MethodSignature Signature
                             marshalAs = parameterMetadataArray[parameterIndex++].MarshalAsDescriptor;
                         }
 
-                        TypeDesc nativeReturnType = MarshalHelpers.GetNativeMethodParameterType(delegateSignature.ReturnType, null, _interopStateManager, true, isAnsi);
+                        TypeDesc nativeReturnType = GetNativeMethodParameterType(delegateSignature.ReturnType, null, _interopStateManager, true, isAnsi);
```

Comment:
Always use named parameter (or at least comments) when you have these bool parameters. Even better, use flag enums.